bodhi/server/views/generic.py

```diff
@@ -68,6 +68,71 @@ def get_latest_updates(request, critpath, security):
     return query.limit(5).all()
 
 
+def get_update_counts(request, releaseid):
+    db = request.db
+    query = db.query(models.Update)
```

Comment:
You could cut these two lines down to one with this:

```
query = request.db.query(models.Update)
```

You could also move it to be below the ```release =``` line below and go with:

```
basequery = request.db.query(models.Update).filter(models.Update.release == release)
```

I'm not actually a huge advocate for crazy one-liners, but I think this is still pretty readable and it cuts down on the code a bit and I think actually simplifies it.